### PR TITLE
fix(Core): Online status of accounts

### PR DESCRIPTION
**Pull request by Deetribs**
The status of the accounts in the database is always "online" after disconnection.

##### TESTS PERFORMED:

Yes, everything fine.

##### HOW TO TEST THE CHANGES:

Log out of your account, the online column will change to 0 after one minute :)

##### Target branch(es):

Master

### CODE REVIEW

File: src/server/game/Server/WorldSession.cpp [hunk 1]
```diff
@@ -126,7 +126,7 @@ WorldSession::WorldSession(uint32 id, WorldSocket* sock, AccountTypes sec, uint8
         m_Address = sock->GetRemoteAddress();
         sock->AddReference();
         ResetTimeOutTime(false);
-        LoginDatabase.PExecute("UPDATE account SET online = %u WHERE id = %u;", realmID, GetAccountId());
+        LoginDatabase.PExecute("UPDATE account SET online = 1 WHERE id = %u;", GetAccountId());
```
Comment (FrancescoBorzi) [inline src/server/game/Server/WorldSession.cpp]: was this suppose to track in which realm the account is online?

**Comment by FrancescoBorzi** [comment]
thanks @Deetribs for your contribution

File: src/server/game/Server/WorldSession.cpp (hunk 1 above)
Comment (Deetribs) [reply src/server/game/Server/WorldSession.cpp]: I don't think it serves to know in which realm the account is connected. But it's true we can keep that.
https://github.com/TrinityCore/TrinityCore/blob/master/src/server/game/Server/WorldSession.cpp#L146

**Comment by Stoabrogga** [comment]
tested successfully :+1:

**Pull request merged**